servicetalk-concurrent-jdkflow/src/test/java/io/servicetalk/concurrent/jdkflow/tck/AbstractCompletableOperatorTckTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright © 2018 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.concurrent.jdkflow.tck;
+
+import io.servicetalk.concurrent.api.Completable;
+import io.servicetalk.concurrent.api.Publisher;
+
+/**
+ * Abstract base class for testing operators provided by {@link Completable} for compliance with the
+ * <a href="https://github.com/reactive-streams/reactive-streams-jvm/tree/v1.0.3/tck-flow">
+ *   Reactive Streams TCK for JDK Flow</a>.
+ * <p>
+ * If you need the flexibility to create the {@link Completable} by yourself you may need to extend
+ * {@link AbstractCompletableTckTest} directly.
+ */
+public abstract class AbstractCompletableOperatorTckTest extends
```

Comment:
None of the TCK tests are required here as JDK flow does not provide a TCK and these tests are just to verify compliance to the TCK. As we are already verifying compliance in the reactivestreams module, doing it here is redundant.

Reply:
I thought this TCK was intended for flow https://github.com/reactive-streams/reactive-streams-jvm/blob/v1.0.3/tck-flow/README.md . Sure will remove it.